fhir2dataset/api_caller.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, url: str, token: str = None):
         self.url = url
         self.status_code = None
         self.results = None
+        self.total = None
         self.next_url = url
```

Comment:
add a Response classsw with these attributes

Reply:
Yeah thank you it was a brillant idea !